src/ert/gui/model/snapshot.py

```diff
@@ -221,12 +221,17 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int) -> None:
                 job_node.data.update(job)
                 if (
                     "current_memory_usage" in job
+                    and "current_memory_usage" in job
                     and job["current_memory_usage"] is not None
                 ):
                     cur_mem_usage = int(float(job["current_memory_usage"]))
                     real_node.data.current_memory_usage = cur_mem_usage
                     self.root.data.current_memory_usage = cur_mem_usage
-                if "max_memory_usage" in job and job["max_memory_usage"] is not None:
+                if (
+                    "max_memory_usage" in job
+                    and "max_memory_usage" in job
```

Comment:
???

Reply:
This test is not possible to simplify with get?

Reply:
it is possible to use get safely, but the typechecker does not understand it.